### PR TITLE
[WIP] Add comma separated multi labels facility

### LINKED ISSUE #32: Posibility to pass multiple values to an action
It would be great to be able to send multiple values in the same action, i.e:

`Derek add label: UX, enhancement, proposal` (`,` as separator, more natural)
or
`Derek add label: UX || enhancement || proposal` (`||` as separator)

I can work on this if the proposal is accepted

### PR DESCRIPTION
**Pull request by rgee0**
Fixes #32

**Currently WIP as further testing is required**

Currently to add multiple labels a maintainer must add one comment for each label.
This change enables multiple labels to be applied through a single comment, by allowing
for a comma separated list to be provided.

As the deletion of multiple labels is currently an iterative process, due to how the github API is presented, an error could occur mid-flow when deleting and cause subsequent labels to not be applied.  Mitigation against this is that labels are visually verifiable.

Signed-off-by: rgee0 <richard@technologee.co.uk>

### CODE REVIEW

**Comment by alexellis** [comment]
How is this WIP? Is there a checklist of things left to do?

File: commentHandler.go [hunk 1]
```diff
@@ -104,36 +104,72 @@ func findLabel(currentLabels []types.IssueLabel, cmdLabel string) bool {
 	return false
 }
 
+func classifyLabels(currentLabels []types.IssueLabel, labelAction string, labelValue string) ([]string, []string) {
+
+	var goodLabels, badLabels []string
```
Comment (alexellis) [inline commentHandler.go]: what is a good/bad label?

Comment (rgee0) [reply commentHandler.go]: The good list is those that can have a valid action performed, the bad list is those that cant (so bad would be duplicates in the case of `add` or absentees in the case of `remove`)

**Comment by rgee0** [comment]
Its WIP as I was going to look at adding an further test which would cover part of `manageLabels()`

File: commentHandler.go (hunk 1 above)
Comment (alexellis) [reply commentHandler.go]: I don't like the choice of wording - good/bad are too loaded. Please could you try to come up with an alternative suggestion?

Comment (rgee0) [reply commentHandler.go]: 'Good' and 'Bad' are now 'Applicable' and 'Unnecessary'

File: commentHandler.go [hunk 2]
```diff
@@ -104,36 +104,72 @@ func findLabel(currentLabels []types.IssueLabel, cmdLabel string) bool {
 	return false
 }
 
+func classifyLabels(currentLabels []types.IssueLabel, labelAction string, labelValue string) ([]string, []string) {
+
+	var applicableLabels, unnecessaryLabels []string
+
+	labelValue = strings.Replace(labelValue, " ", "", -1)
```
Comment (rgee0) [inline commentHandler.go]: This needs changing.  What if the labelValue has contains a space, such as 'help wanted'?

Use TrimSpace in the range loop instead?

**Pull request closed without merging**